### PR TITLE
translation in French (needs proof checking!)

### PR DESCRIPTION
**Pull request by leojpod**
Hi there, 

Here is a quick attempt at translating that into French. 
Is there another French around to check it?

### CODE REVIEW

File: src/Http/Error/View/French.elm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+errorToFrench : Http.Error -> String
+errorToFrench error =
+    case error of
+        BadUrl url ->
+            "L'URL fournie n'est pas valide : " ++ url ++ "."
+
+        Timeout ->
+            "Le serveur n'as pas répondu à temps (la requête a expirée)."
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: `(la requête a expiré)`
Pas de `e` supplémentaire

File: src/Http/Error/View/French.elm [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                ++ errorMessage
+                ++ "\" et "
+                ++ responseToFrench response
+
+
+responseToFrench : Http.Response String -> String
+responseToFrench response =
+    "La tentative de connexion à "
+        ++ response.url
+        ++ " a renvoyée le code d'erreur suivant : "
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: Idem ici, pas `renvoyée` mais `renvoyé`

File: src/Http/Error/View/French.elm [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        ++ toString response.status.code
+        ++ " qui correspond à : \""
+        ++ response.status.message
+        ++ "\"."
+
+
+{-| Transform un Http.Error en une vue Html
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: `Transforme`

File: src/Http/Error/View/French.elm [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+    view (BadUrl "blob")
+        --> Html.text "L'URL fournie n'est pas valide : blob"
+
+    view Timeout
+        --> Html.text "Le serveur n'as pas répondu à temps (la requête a expirée)."
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: `expirée` -> `expiré`

File: src/Http/Error/View/French.elm [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+    view NetworkError
+        --> "Impossible d'établir une connexion. Votre réseau fonctionne-t-il?"
+
+    view (BadStatus { url = "http://blob.com", status = { code = 500, message = "oops"}})
+        --> Html.text "La tentative de connexion à http://blob.com"
+        --> " à renvoyer le code d'erreur suivant : 500"
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: `à renvoyer` -> `a renvoyé`

File: src/Http/Error/View/French.elm [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        --> " qui correspond à : \"oops\""
+
+
+    view (BadPayload (("no such field name")) { url = "http://blob.com", status = { code = 505, message = "oops"}})
+        --> Html.text "L'erreur suivante s'est produite:  no such field name"
+        --> " et La tentative de connexion à http://blob.com"
+        --> " à renvoyer le code d'erreur suivant : 505"
```
Comment (mpizenberg) [inline src/Http/Error/View/French.elm]: `à renvoyer` -> `a renvoyé`

File: src/Http/Error/View/French.elm (hunk 1 above)
Comment (pauldijou) [reply src/Http/Error/View/French.elm]: `Le server n'a pas` (pas de `s`)

File: src/Http/Error/View/French.elm (hunk 4 above)
Comment (pauldijou) [reply src/Http/Error/View/French.elm]: `n'as` -> `n'a`

File: src/Http/Error/View/French.elm (hunk 6 above)
Comment (pauldijou) [reply src/Http/Error/View/French.elm]: `s'est produite:` -> `s'est produite :`
`et La` -> `et la`

File: src/Http/Error/View/French.elm (hunk 1 above)
Comment (leojpod) [reply src/Http/Error/View/French.elm]: thx

**Comment by leojpod** [comment]
@eeue56 It should ready now :)
Thanks @pauldijou & @mpizenberg !

**Comment by eeue56** [comment]
Thank you! I'll merge and publish soon!

**Comment by leojpod** [comment]
cheers!

**Pull request merged**